metadata/src/test/java/org/apache/kafka/controller/PartitionReassignmentRevertTest.java

```diff
@@ -78,7 +79,7 @@ public void testSomeAdding() {
             setAddingReplicas(new int[]{4, 5}).setLeader(3).setLeaderRecoveryState(LeaderRecoveryState.RECOVERED).setLeaderEpoch(100).setPartitionEpoch(200).build();
         PartitionReassignmentRevert revert = new PartitionReassignmentRevert(registration);
         assertEquals(Arrays.asList(3, 2, 1), revert.replicas());
-        assertEquals(Arrays.asList(2), revert.isr());
+        assertEquals(Collections.singletonList(2), revert.isr());
```

Comment:
I think keeping the multiple assert similar is more readable so you may want to keep `Arrays.asList()` here